lib/cocina_display/concerns/forms.rb

```diff
@@ -47,6 +47,23 @@ def genres_search
         end.uniq
       end
 
+      # All map-related data to be rendered for display as a single value.
+      # Includes map scale, projection info, and geographic coordinate subjects.
+      # @return [Array<DisplayData>]
+      def map_display_data
```

Comment:
Should MapData be it's own concern because it's combining metadata from both form and subject?

Reply:
I wish this was easier to answer. The concerns aren't fully encapsulated; there are a few instances where they call each other (e.g. the sort contributor name has to use the sort title): https://github.com/sul-dlss/cocina_display/blob/528c240bb9818ef3ba1d862a6ea453846e47902c/lib/cocina_display/concerns/contributors.rb#L77

If anything, maybe we could have a `Display` or `DisplayData` concern for all of the `_display_data` methods? I really don't know what the best choice is.